packages/dashboard-core-plugins/src/panels/IrisGridPanel.jsx

```diff
@@ -123,6 +123,7 @@ export class IrisGridPanel extends PureComponent {
       pluginFetchColumns: [],
       modelQueue: [],
       pendingDataMap: new Map(),
+      frozenColumns: [],
```

Comment:
We should be initializing to the `frozenColumns` prop, which should be `null` by default.

Reply:
Agreed.